src/main.rs

```diff
@@ -2,164 +2,44 @@
 // Usage: commp <path to file> [fp]
 // specify "fp" to run through filecoin_proofs
 
-use std::cmp;
-use std::convert::TryFrom;
+mod commp;
+
 use std::env;
 use std::fs::File;
-use std::io;
-use std::io::{Cursor, Read, Seek, SeekFrom};
 
-use anyhow::ensure;
+#[macro_use]
+extern crate log;
```

Comment:
Remove those to lines and instead import the logging macros directly in the modules where you use them (`use log::{debug, info}`).